client/src/Game/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import '../App.css';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import { saveAs } from 'file-saver';
+import './literallycanvas.css';
+import Banner from '../Banner';
+import { withAuthorization } from '../Session';
+import { withFirebase } from '../Firebase';
+import { compose } from 'recompose';
+import * as ROUTES from './constants/routes';
```

Comment:
Path is incorrect, should be '../constants/routes'

Reply:
fixed